server/index.js

```diff
@@ -1,11 +1,20 @@
-const db = require('../db/index.js');
+// const db = require('../db/index.js');
 const express = require('express');
 const app = express();
 const port = 3000;
+var cors = require('cors')
 
-app.get('/', (req, res) => {
+app.use(cors());
+app.use(express.static('./public'));
+
+app.get('/api/reviews:id', async (req, res) => {
+  const data = db.get(req.params.id);
+  // console.log(res);
+  res.send(data).status(200);
+});
```

Comment:
@drtcxrch - Interesting! I don't think I've seen it before where the url param is directly attached. Does this work as-is? Does `req.params.id` give you the right data?
Just for your own sanity (particularly on the `App` side of things when requesting data from your own server) it may be worth refactoring to `/api/reviews/:id`, but if this is working as intended, you don't necessarily need to change it.

Reply:
Haha, so I screwed up with the whole feature branch ordeal and seemed to have deleted some work in the process, including my functional api calls. I've fixed it now though and gonna push my work. So, no that code doesn't work.

Reply:
@drtcxrch I just did the same actually. 😂 Learning process, this.